packages/dd-trace/src/appsec/sdk/utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+function getRootSpan (tracer) {
```

Comment:
there is another and more supported way to get a web root span:
`const rootSpan = web.root(req)`
Could that be used instead of digging into the spanContext ?

Reply:
But it requires the the request object. Should we request it to track events?
pro: rootSpan always available, not context dependant
cons: we need request as parameter

Reply:
For example, `setUser` method doesn't require it